package.json

```diff
@@ -23,16 +46,18 @@
     "expo-status-bar": "1.0.4",
     "expo-updates": "0.8.5",
     "formik": "2.2.9",
+    "jest": "^27.1.1",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-41.0.0.tar.gz",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-42.0.0.tar.gz",
```

Comment:
this slipped in somehow, should be fine though?

Reply:
It seeme to be